elements/pl-checkbox/pl-checkbox.py

```diff
@@ -426,7 +434,7 @@ def grade(element_html, data):
         else:
             raise ValueError(f'Unknown value for partial_credit_method: {partial_credit_method}')
 
-    data['partial_scores'][name] = {'score': score, 'weight': weight}
+    data['partial_scores'][name] = {'score': score, 'weight': weight, 'feedback': feedback}
```

Comment:
Can you explain why this is being stored on `partial_scores`? I think all the feedback is already stored in `data['params'][name][*]['feedback']`, and I'd rather keep that as the "canonical" source for feedback if possible.

Reply:
See https://github.com/PrairieLearn/PrairieLearn/pull/4656#issuecomment-899773221

Reply:
Super helpful - thanks! Jonatan's comment on the other PR for `pl-multiple-choice` makes a lot of sense.